### PR TITLE
ISSUE-192 Specify com:linagora:params:jmap:pgp extension

### PR DESCRIPTION
**Pull request by chibenwa**
Along the way I do fix some tiny some JMAP spec compliance issues...

### CODE REVIEW

**Comment by chibenwa** [comment]
Cc @vttranlina fixed some stuff here :-p

File: tmail-backend/jmap/extensions/doc/jmapGpgExtension.md [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+This extension allow access to encrypted email subpart for efficient email display, typically within a webmail. The clients
+need to access part of the underlying data and do not need to perform any MIME parsing / manipulation.
+
+We thus introduce the following objects:
+
+ - `EncryptedEmailFastView` holds information required to list emails.
+ - `EncryptedEmailDetailedView` holds information required to view emails.\
+
+Furthermore, this extension allows clients to upload, list and remove GPG pulic keys used to encrypt their mailbox content.
+
```
Comment (vttranlina) [inline tmail-backend/jmap/extensions/doc/jmapGpgExtension.md:12]: pulic -> public

**Review by vttranlina — APPROVED** [review]

File: tmail-backend/jmap/extensions/doc/jmapGpgExtension.md [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+This extension allow access to encrypted email subpart for efficient email display, typically within a webmail. The clients
+need to access part of the underlying data and do not need to perform any MIME parsing / manipulation.
+
+We thus introduce the following objects:
+
+ - `EncryptedEmailFastView` holds information required to list emails.
+ - `EncryptedEmailDetailedView` holds information required to view emails.\
+
+Furthermore, this extension allows clients to upload, list and remove GPG pulic keys used to encrypt their mailbox content.
```
Comment (chibenwa) [inline tmail-backend/jmap/extensions/doc/jmapGpgExtension.md:11]:
```suggestion
Furthermore, this extension allows clients to upload, list and remove GPG public keys used to encrypt their mailbox content.
```

File: tmail-backend/jmap/extensions/doc/jmapGpgExtension.md [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+
+Servers supporting the JMAP GPG extension advertise it via the `com:linagora:params:jmap:pgp` capability.
+
+The value of this property in the JMAP session capabilities property is an empty object.
+
+The value of this property in an account’s accountCapabilities property is an empty object.
+
+## Keystore
+
+The `Keystore` object represent an underlying GPG public key
```
Comment (Arsnael) [inline tmail-backend/jmap/extensions/doc/jmapGpgExtension.md:23]:
```suggestion
The `Keystore` object represents an underlying GPG public key
```

File: tmail-backend/jmap/extensions/doc/jmapGpgExtension.md [hunk 4]
```diff
@@ -0,0 +1,501 @@
...
+
+A `Keystore` have the following fields:
```
Comment (Arsnael) [inline tmail-backend/jmap/extensions/doc/jmapGpgExtension.md:25]:
```suggestion
A `Keystore` has the following fields:
```

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
Just squashed fixups

**Review by vttranlina — APPROVED** [review]

**Pull request merged**